static/js/src/advantage/offers/Offers.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { Row, Col } from "@canonical/react-components";
+import OffersList from "./components/OffersList";
+
+const Offers = () => {
+  return (
+    <div className="p-strip--suru-topped">
+      <Row>
+        <Col size={12}>
+          <h1>Your Ubuntu Advantage Offers</h1>
```

Comment:
```suggestion
          <h1>Your Ubuntu Advantage offers</h1>
```

Reply:
I followed the case on the wireframe. I did find it a bit odd to capitalise "offers" @wgx ?